draft-ietf-rats-uccs.md

```diff
@@ -475,6 +475,30 @@ an UCCS by enclosing it with a tag number TBD601:
 <!--  LocalWords:  Verifier's CWTs Attester Verifier FCFS
  -->
 
+# JSON Support
+
+The above definitions, concepts and security considerations all may be applied to define a JSON-encoded unsigned token.
+This can be referred to as a "UJCS", an Unprotected JWT Claims Set".
+The CDDL in the appendix above works for JSON as well as CBOR through the use of the JC<> CDDL generic.
+The definition for a UJCS token in CDDL is identical to that of a Claims-Set.
```

Comment:
I am having a some difficulties parsing that sentence.

Reply:
I'm not sure what the issue is with parsing here.

_"A UJCS token is a Claims-Set with JSON encoding"_ would sound better?

Reply:
Neither UCCS nore "UJCS" are token?

Maybe: The CDDL definition in Figure 1 can be used for "UJCS".

Reply:
> Neither UCCS nore "UJCS" are token?

ok, now I can parse your parsing error :-)

> Maybe: The CDDL definition in Figure 1 can be used for "UJCS".

yeah, why not.

Reply:
```suggestion
The definition for a UJCS token in CDDL is identical to that of a Claims-Set.
```